.travis.yml

```diff
@@ -9,7 +9,7 @@ matrix:
   include:
     - env: LANGUAGE=ansible       VERSION=2.3.0.0-r1 PYTHON_VERSION=2.7.14-r0
     - env: LANGUAGE=aws           VERSION=1
-    - env: LANGUAGE=dind-aws      VERSION=1    DOCKER_COMPOSE_VERSION=1.21.2 GLIBC_VERSION=2.23-r3
+    - env: LANGUAGE=dind-aws      VERSION=1    DOCKER_COMPOSE_VERSION=1.22.0 GLIBC_VERSION=2.23-r3
```

Comment:
Can you update the CHANGELOG.md please?

Reply:
Fixed, but I see that ansible build always fail, even in previous merge ...is it normal ?

Reply:
Yep please replace 08 by 09, and about the failing build, it's not normal :), can you have a look please?

Reply:
@mremi fixed by replacing python version in ansible build